security policy (#1738)
fastify/fastify: SECURITY.md

# Security Policy

This document describes the management of vulnerabilities for the Fastify project.

## Reporting vulnerabilities

> Individuals who find potential vulnerabilities in Fastify are invited to complete a vulnerability report on the dedicated HackerOne organization: [https://hackerone.com/nodejs-ecosystem](https://hackerone.com/nodejs-ecosystem) for the modules of Node.js

Reviewer: ```suggestion
Individuals who find potential vulnerabilities in Fastify are invited to complete a vulnerability report via the dedicated HackerOne tool for Node.js modules: [https://hackerone.com/nodejs-ecosystem](https://hackerone.com/nodejs-ecosystem).
```

### How to report a vulnerabiliy

> It is of the outmost importance that you read carefully [**HOW TO REPORT A VULNERABILIY**](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md) written by the Security Working Group of Node.js.

Reviewer: ```suggestion
It is of the utmost importance that you read carefully [**HOW TO REPORT A VULNERABILIY**](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md) written by the Security Working Group of Node.js.
```

## Handling vulnerability reports

When a potential vulnerability is reported and confirmed the Fastify Core Team will intervene in the
`follow-up` stage of the process following the workflow and the timings described in the Security WG's document.

### Vulnerabilities found outside this process

> ⚠ Contrarily to the Security WG's document, the Fastify project does not support any reporting outside the HackerOne process.

Reviewer: ```suggestion
⚠ The Fastify project does not support any reporting outside the HackerOne process.
```

## The Fastify Core team

> The core team is responsible for the management of this process.

Reviewer 2: This it would be good to be a link.

> They are responsible for the management of this process.

Reviewer: ```suggestion
The core team is responsible for the management of this process.
```
